src/clients/users_finish.rb

```diff
@@ -71,9 +73,12 @@ def main
 
           # During installation, some package could add a new user, so we
           # need to read them again before writing.
```

Comment:
What I wanted to express is that we need to Export -> Re-read -> Import again (that's the reason to put the comment before the whole block).

But I see that's not clear at all, so I've added some extra documentation.